src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -84,6 +84,11 @@
   -->
   <Import Project="$(MSBuildThisFileDirectory)versioning.targets" Condition="'$(ExcludeVersioningImport)'!='true'" />
 
+  <!--
+    Import the localization target
+  -->
+  <Import Project="$(MSBuildThisFileDirectory)localization.targets" Condition="'$(ExcludeLocalizationImport)'!='true' And Exists('$(MSBuildProjectDirectory)\MultilingualResources\$(MSBuildProjectName).de.xlf')" />
```

Comment:
Yes I was thinking something along those lines. I agree that the readability isn't way better overall but it does make the import condition a simple Boolean value and we can control it in various ways. For example we could explicitly set it to false and override the Exists check on xlf if we needed that flexibility, or change the check to look for a different file in some cases. I think it enables us more flexibility so we don't have to wait on a build tools update to unblock ourselves if we need to.

Also is "de.xlf" the right file to key on? It might be OK for now but we may want to consider *.xlf at some point, although I don't think that works with Exists we could use an item group if we needed that support.